### PR TITLE
Refactor constant notation to use kCamelCase

### PR DESCRIPTION
**Pull request by Harry-Hopkinson**
Refactor more constexpr constant notation to kCamelCase.

### CODE REVIEW

File: src/openrct2/rct1/Tables.cpp [hunk 1]
```diff
@@ -686,13 +686,13 @@ namespace RCT1
 
         if (rct1VehicleType == VehicleType::HeartlineTwisterCars)
         {
-            return vehicleSubEntry == HEARTLINE_TWISTER_FORWARDS ? 0 : 1;
```
Comment (Harry-Hopkinson) [inline src/openrct2/rct1/Tables.cpp:689]: It appears that my formatter has stripped off some blank lines, should I remove this?

Comment (Gymnasiast) [reply src/openrct2/rct1/Tables.cpp:689]: It looks like your formatter has removed some trailing whitespace. That is good, actually.

File: src/openrct2/rct2/Limits.h [hunk 2]
```diff
@@ -13,15 +13,15 @@
...
-    constexpr uint8_t MaxStaff = 200;
-    constexpr uint8_t MaxBanners = 250;
+    constexpr uint8_t kMaxStaff = 200;
+    constexpr uint8_t kMaxBanners = 250;
     constexpr uint8_t kMaxTrainsPerRide = 32;
     constexpr uint8_t kMaxVehicleColours = 32;
     constexpr uint8_t kDowntimeHistorySize = 8;
-    constexpr uint16_t MaxEntities = 10000;
+    constexpr uint16_t kMaxEntities = 10000;
     constexpr uint16_t MaxEntitiesRCTCExtended = 15000; // Used in files marked with “classic flag” 0xF
```
Comment (Gymnasiast) [inline src/openrct2/rct2/Limits.h:22]: Might as well do all constants in this block.

File: src/openrct2/rct1/Limits.h [hunk 3]
```diff
@@ -13,16 +13,16 @@
...
-    constexpr uint32_t Num_LL_CSG_Entries = 69917;
-    constexpr uint32_t LL_CSG1_DAT_FileSize = 41402869;
-    constexpr uint32_t NumTerrainSurfaces = 16;
-    constexpr uint32_t NumTerrainEdges = 15;
+    constexpr uint8_t kMaxMapSize = 128;
+    constexpr uint8_t kMaxStaff = 116;
+    constexpr uint16_t kMaxAnimatedObjects = 1000;
+    constexpr uint8_t kMaxBanners = 100;
+    constexpr int32_t kCoordsZStep = 4;
+    constexpr uint32_t kNumLlCsgEntries = 69917;
```
Comment (Gymnasiast) [inline src/openrct2/rct1/Limits.h:24]: Could you change `Ll` to `LL` in this entry and the next one? It looks so weird :')

Comment (Harry-Hopkinson) [reply src/openrct2/rct1/Limits.h]: Should I do the same for `kLlCsg1DatFileSize`?

File: src/openrct2/rct1/Limits.h [hunk 4]
```diff
@@ -13,16 +13,16 @@
...
-    constexpr uint32_t LL_CSG1_DAT_FileSize = 41402869;
-    constexpr uint32_t NumTerrainSurfaces = 16;
-    constexpr uint32_t NumTerrainEdges = 15;
+    constexpr uint8_t kMaxMapSize = 128;
+    constexpr uint8_t kMaxStaff = 116;
+    constexpr uint16_t kMaxAnimatedObjects = 1000;
+    constexpr uint8_t kMaxBanners = 100;
+    constexpr int32_t kCoordsZStep = 4;
+    constexpr uint32_t kNumLLCsgEntries = 69917;
+    constexpr uint32_t kLlCsg1DatFileSize = 41402869;
```
Comment (Gymnasiast) [inline src/openrct2/rct1/Limits.h:25]: You missed this one.

File: src/openrct2/ride/coaster/CorkscrewRollerCoaster.hpp [hunk 5]
```diff
@@ -23,26 +23,26 @@
...
-    { CORKSCREW_RC_BLOCK_BRAKE_SW_NE_OPEN, CORKSCREW_RC_BLOCK_BRAKE_SW_NE_CLOSED },
-    { CORKSCREW_RC_BLOCK_BRAKE_NW_SE_OPEN, CORKSCREW_RC_BLOCK_BRAKE_NW_SE_CLOSED },
-    { CORKSCREW_RC_BLOCK_BRAKE_SW_NE_OPEN, CORKSCREW_RC_BLOCK_BRAKE_SW_NE_CLOSED },
-    { CORKSCREW_RC_BLOCK_BRAKE_NW_SE_OPEN, CORKSCREW_RC_BLOCK_BRAKE_NW_SE_CLOSED },
+static constexpr uint32_t kCorkscrewRcBlockBrakeSwNeOpen = 16232;
+static constexpr uint32_t kCorkscrewRcBlockBrakeNwSeOpen = 16233;
+static constexpr uint32_t kCorkscrewRcBlockBrakeSwNeClosed = 16234;
+static constexpr uint32_t kCorkscrewRcBlockBrakeNwSeClosed = 16235;
+
+static constexpr uint32_t _kCorkscrewRcBlockBrakeImages[NumOrthogonalDirections][2] = {
```
Comment (Gymnasiast) [inline src/openrct2/ride/coaster/CorkscrewRollerCoaster.hpp:31]: This should not have a `_` prefix.

File: src/openrct2/rct2/Limits.h (hunk 2 above)
Comment (Gymnasiast) [reply src/openrct2/rct2/Limits.h:22]: Why did you mark this as resolved? You haven’t addressed it yet.

File: src/openrct2/rct2/Limits.h [hunk 6]
```diff
@@ -13,23 +13,23 @@
...
+    constexpr uint16_t kMaxAnimatedObjects = 2000;
+    constexpr uint8_t kMaxResearchedRideTypeQuads = 8;  // With 32 bits per uint32_t, this means there is room for
+                                                        // 256 types.
+    constexpr uint8_t kMaxResearchedRideEntryQuads = 8; // With 32 bits per uint32_t, this means there is room for
+                                                        // 256 entries.
+    constexpr uint8_t kMaxResearchedSceneryItemQuads = 56;
+    constexpr const uint16_t kMaxResearchedSceneryItems = (kMaxResearchedSceneryItemQuads * 32); // There are 32
+                                                                                                 // bits per
+                                                                                                 // quad.
     constexpr uint16_t MaxResearchItems = 500;
```
Comment (Gymnasiast) [inline src/openrct2/rct2/Limits.h:33]: There is still a list of unaddressed constants here.

File: src/openrct2/rct2/Limits.h [hunk 7]
```diff
@@ -13,37 +13,37 @@
...
+    constexpr uint8_t kMaxLargeSceneryObjects = 128;
+    constexpr uint8_t kMaxWallSceneryObjects = 128;
+    constexpr uint8_t kMaxBannerObjects = 32;
+    constexpr uint8_t kMaxPathObjects = 16;
+    constexpr uint8_t kMaxPathAdditionObjects = 15;
+    constexpr uint8_t kMaxSceneryGroupObjects = 19;
+    constexpr uint8_t kMaxParkEntranceObjects = 1;
+    constexpr uint8_t kMaxWaterObjects = 1;
+    constexpr uint8_t kMaxScenarioTextObjects = 1;
     constexpr uint8_t RideTypeCount = 91;
```
Comment (Harry-Hopkinson) [inline src/openrct2/rct2/Limits.h:47]: This variable is unused, should I remove it?

File: src/openrct2/rct2/Limits.h (hunk 2 above)
Comment (Harry-Hopkinson) [reply src/openrct2/rct2/Limits.h]: Sorry, I thought block meant the top constants clumped and not all of the constants. I have added the ones that I have missed.

File: src/openrct2/rct2/Limits.h (hunk 7 above)
Comment (Harry-Hopkinson) [reply src/openrct2/rct2/Limits.h]: I have removed it.

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**